tcmu-runner.service

```diff
@@ -4,5 +4,5 @@ Description=LIO Userspace-passthrough daemon
 [Service]
 Type=dbus
 BusName=org.kernel.TCMUService1
+KillMode=process
```

Comment:
OKAY - I do not see any reason why this should be restarted in case if it exist with a failure.
At-least I cannot track if tcmu-runner had restarted in between or running non stopped (With no hacks or dirty scripts). The auto restart still makes many reasons for failure hidden, at-least for me  if I notice an abnormal stop/exit of the daemon, I could have investigate the cause and try to fix it a bit.

@mikechristie Have we discussed the data inconsistency that an auto restart bring in ?
do you remember the exact scenario ?